### DISCUSSION TITLE
Why isn't this child machine receiving an event?

### DISCUSSION THREAD

**Discussion by jackharding** [statelyai/xstate#1833] [Q&A] [answered]
Here's a [sandbox](https://codesandbox.io/s/hidden-voice-dd99n).

I don't understand why the coundownMachine doesn't receive the INCREASE_TIME event.

**Comment by Andarist** [comment, marked as answer]
You have a bug in your schema. Instead of this:
```js
INCREASE_TIME: assign<Context, any>({ /* ... */ })
```
you should declare this:
```js
INCREASE_TIME: {
    actions: assign<Context, any>({ /* ... */ })
}
```

Could you report this as an issue? I believe we could do a better job at surfacing this kind of problems to users.

**Reply by jackharding** [reply]
I'm a moron. Thanks! Will do.

**Comment by danielkcz** [comment]
TypeScript kinda did catch the problem, but it's obviously less clear from that message what's wrong.

![image](https://user-images.githubusercontent.com/1096340/104439387-56370f80-5591-11eb-96cf-6514040ed7d0.png)

I would highly recommend you to have a look at https://github.com/mattpocock/xstate-codegen that heavily improves typescript experiences with machines. The first benefit is that you don't to manually sync the `StateSchema` and a problem like this is shown more clearly.
